changelog/2024.2.0.rst

```diff
@@ -18,6 +18,29 @@ ESPHome 2024.2.0 - 21st February 2024
     Honeywell HIH I2C, components/sensor/honeywell_hih_i2c, honeywellhih.jpg
     Combination, components/sensor/combination, function.svg, dark-invert
 
+micro Wake Word
+---------------
+
+ESPHome implements an on-device wake word detection framework from `microWakeWord <https://github.com/kahrendt/microWakeWord>`__.
+See the :doc:`component documentation </components/micro_wake_word>` for details on how to use it.
+
+Check out the live stream replay below for more details about how it works and came to be.
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/NQIv3nsu7dE"
+            title="YouTube video player" frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen>
+    </iframe>
```

Comment:
I'm unclear on what the linter is complaining about here; we have multiple other instances of this exact code and it's fine.